### ISSUE TITLE
false positive no-member on numpy imports

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#58]
Originally reported by: **Trey Greer (BitBucket: [trey_greer](http://bitbucket.org/trey_greer))**

---

With pylint 1.0.0, astroid 1.0.0, common 0.60.0,  and python 3.3.2 I'm getting no-member errors on all imports from numpy.

```
#!python
''' demonstrate numpy import false positive '''
import numpy

A1 = numpy.array([1, 2, 3])
print(A1)
```

gives

```
#!

No config file found, using default configuration
************* Module test
E:  4, 5: Module 'numpy' has no 'array' member (no-member)


```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/58

**Comment by gergelypolonkai** [comment]
I still get this error, even using `--extension-pkg-whitelist=numpy` (or its RC file counterpart). Adding `unsafe-load-any-extension=yes` also doesn’t help.

I’m using pylint 1.7.1 (from PyPI) and numpy 1.12.1, if it matters.

Here is a minimal example to reproduce (I filtered out docstring-relate warnings for readability’s sake):

``` python
import numpy

def my_func():
    var = 1

    return isinstance(var, numpy.float64)
```

```
$ pylint --extension-pkg-whitelist=numpy test
************* Module test
E: 14,20: Module 'numpy' has no 'float64' member (no-member)
```

**Comment by gergelypolonkai** [comment]
Sorry for the noise, it seems this is already documented in #779
